src/Arriba/Arriba/Server/Authorization/ArribaIdentityNormalizer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using System.Security.Claims;
+using System.Security.Principal;
+using Arriba.ParametersCheckers;
+
+namespace Arriba.Server.Authorization
+{
+    public class ArribaIdentityNormalizer : IArribaIdentityNormalizer
+    {
+        private const string ClaimTypeName = "name";
+
+        public ClaimsPrincipal NormalizeIdentity(ClaimsPrincipal user, bool throwExceptionWhenNameNotFound = true)
+        {
+            user.ThrowIfNull(nameof(user));
+            ParamChecker.ThrowIfNull(user.Identity, nameof(user.Identity));
```

Comment:
seems like this line could be for consistency
```suggestion
            user.Identity.ThrowIfNull(nameof(user.Identity));
```

Reply:
We don't have a extension for the Identity, but the ParamChecker has the thowIfNull using generics.
I was in doubt at the time to create an extension or use the generic one, at the end I preferred to use the generic because the others have a higher use frequency.

Reply:
I added a guard method for IIdentity